content/departments/engineering/teams/growth/cloud-trial-operations.md

```diff
@@ -1,50 +1,169 @@
+  - Talk to an Account Executive and Customer Engineer (CE) to get help setting up a self-hosted instance.
+  - Talk to an Account Executive and CE to get a cloud instance trial (hereafter called "a CE-requested trial"). Those instances can be setup to match customer needs (scale, )
+- **(NEW!) Signup for a self-serve trial from signup.sourcegraph.com** (hereafter called "self-serve trial"). This page zooms in only on this option.
+
+> NOTE: This Handbook page describes the workflow **for self-serve trials only**.
+
+Most call to actions (CTAs) on Sourcegraph sites and posts will either let users request a PLG trial, or contact us to discuss one of the other options.
 
 ## Context
 
-Cloud is the default way of deploying Souregraph, but there are some limits to delivering a cloud trial instance today.
+Cloud is the default way of deploying Sourcegraph, but there are some limits to delivering a cloud trial instance today.
 
 - There is a cap on the number of instances we can managed at any given time ([SLA](../../../cloud/index.md#slas-for-managed-instances)). Therefore, when someone requests an instance, we have to qualify them to make sure that we have capacity and that a cloud instance is the best option to meet their needs.
-- Cloud instances take some time to provision ([SLA](../../../cloud/index.md#slas-for-managed-instances)), that makes it so signing up for a cloud instance is not instant.
+- Cloud instances take some time to provision ([SLA](../../../cloud/index.md#slas-for-managed-instances)), that makes it so signing up for a cloud instance is not instant. To work around this limitation, we pre-provision instances in some cases, with the trade-off that they have random subdomains (eg. `hjbkn.sourcegraph.com`).
 - The workflow for qualifying a user that requests a trial instance, provisioning the instance, handing it off to the requesting user, and helping them onboard is very manual. This last bit will be automated in the short term (next few months).
+- The onboarding on Sourcegraph instances is very rudimentary. The Product Growth team is working on that next.
+- We aim to give all self-serve trials a white-glove experience through custom onboarding emails, sessions with Customer Engineers, etc. Most of this will eventually be automated.
 
-## Expectations from stakeholders
+## Workflow
+
+Here is an overview of the workflow and the role of any and all Sourcegraph teams.
+
+TLDR:
+
+- We maintain a list of prequalified target accounts, that will instantly get an instance in-product
+- All others go through manual qualification by inbound SDRs, and if qualified, an instance is provisioned by the Cloud team and handed-off to the reqester by a CE
+- Instance hand-off (if not automated), white-glove onboarding, and trial extension/termination/conversion are owned by the CE assigned to the Opportunity.
+- Trials should be terminated after 15 days if there's no activity on the instance, and extented to 30 days if there's any activity.
```

Comment:
Do we consider all trials for 15days initialy? Or only PLG ones?

Reply:
Only PLG ones. Does that work?